core/interfaces/i_focusable_node.ts

```diff
@@ -59,6 +59,9 @@ export interface IFocusableNode {
    * they should avoid the following:
    * - Creating or removing DOM elements (including via the renderer or drawer).
    * - Affecting focus via DOM focus() calls or the FocusManager.
+   *
+   * Implementations should consider scrolling themselves into view here; that
```

Comment:
```suggestion
   * Implementations may consider scrolling themselves into view here; that
```

We probably don't want to be prescriptive here since, as I understand it, most nodes don't actually care to do this.

Edit: Ah, looking at #9288 it's more obvious that many nodes need to do this. I'm fine with keeping the current terminology if you prefer, but I think other make sense.

Ron may also have thoughts on this wording.

Reply:
Done.